### DISCUSSION TITLE
Allow fullstop/dot in config names

### DISCUSSION THREAD

**Discussion by andymanning** [Dynatrace/dynatrace-configuration-as-code#332] [Archive] [answered]
We use a dot notation for our entity names eg.
mobileapp.maintenance.window

When these are download with monaco the . is removed and it becomes mobileappmaintenancewindow

I've found the code that governs the naming and have created a local version which does allow the dot in the names. It works on both Linux (Ubuntu) and Windows.  

The change is to allow the regexp to allow dot (.) 

**file**: afero_utils.go
**func**: func SanitizeName(name string) string 

line _reg, err := regexp.Compile("[^a-zA-Z0-9-]+")_ 
becomes : _reg, err := regexp.Compile("[^a-zA-Z0-9-.]+")_

Is this a good idea?

**Comment by tatomir146** [comment, marked as answer]
@andymanning thanks for reporting this. This will generally be changed once the monaco config 2.0 is implemented. There is general overhaul of download functionality coming.
